.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,38 @@
+<!-- Thanks for contributing to our project! We appreciate your time and effort.
+Please fill out the information below to expedite the review and merge of your pull request.
+-->
+
+#### What this PR does / why we need it:
+<!-- A clear description of the problem you are trying to solve -->
+
+
+#### Changes made
+<!-- Outline the specific changes made in this merge request. -->
+
+
+#### Which issue(s) this PR fixes:
+<!--
+Any reference to relevant issue(s).
+Please use the following format, so that the issue will be automatically closed when this PR is merged (see https://help.github.com/articles/closing-issues-using-keywords/)
+
+`Fixes #<issue number>`
+
+If there is not a correspondent issue yet, you might want to open a new one yourself, describing the problem you observed.
+-->
```

Comment:
Suggesting a macro for optional subsections would be nice such as NONE for when there isn't an issue to fix. For us in Medik8s, it could be nice to add here a link to Jira (i.e., `ECOPROJECT-XXXX`).

Reply:
What do you mean by "macro"?

Reply:
I meant that if there is no needed test for the PR we will still have the _Test plan_ section, then we can use a uniform macro (e.g., **none**/? ) as this PR has no test and a new test would be redundant (e.g. for creating a new optional makefile target). I am not sure how helpful it is but it might be more user-friendly for someone inexperienced and not sure what to write, so he could just use this macro. While we review the PR we could guide him better on what to write and etc.

Reply:
I think I understand now, and while I agree that some changes might not need new tests, it is still necessary that _old_ tests pass.

Reply:
I mean, the Template section about Tests also says

> (ensure) Existing Unit and E2E tests are passing

and I believe it is ok to use the section to state that "no tests are needed" (which we can later discuss in the PR review), but I don't really want to encourage people to leave PRs without even thinking about making tests :)